sherpa-onnx/csrc/offline-lm-config.h

```diff
@@ -19,14 +19,21 @@ struct OfflineLMConfig {
   int32_t lm_num_threads = 1;
   std::string lm_provider = "cpu";
 
+  // LODR
+  std::string lodr_fst = "";
```

Comment:
```suggestion
  std::string lodr_fst;
```

`std::string` is default initialized. There is no need to use `""` to initialize it.